### PR TITLE
side-menu nav design update

### LINKED ISSUE #203: [Enhancement] Redesigning side menu on toggle in small screens
- As of now the side-menu on small screens look a bit odd, so I would like to redesign it to give it a better and smooth look.

- I am a GSSoC23 contributor so please assign this issue under gssoc

### How it looks now :

image

### How I plan to make it :

image

#### *Any kind of feedback is appreciated

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Related Issue

Closes: #203 

### Describe the changes you've made

- Updated the side-menu Nav bar design to make it look more smooth and user-friendly

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

- I tested my changes in brave, edge, and chrome responsive window which allows to check on various mobile screen resolutions.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### Original
image

### Updated
 image |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/UniKonf/vibey/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by ujjawaltyagii** [comment]
Please review this PR under GSSoC and add labels too ! @Deepu178  @J-Harsh

File: src/styles/globals.css [hunk 1]
```diff
@@ -123,3 +123,20 @@ body {
 .ais-SearchBox input {
   @apply h-full w-full rounded-xl border-2 border-primary/30 bg-transparent p-4 text-3xl outline-none placeholder:text-base-content focus:ring-2 focus:ring-primary;
 }
+
```
Comment (Deepu178) [inline src/styles/globals.css]: @ujjawaltyagii We are using tailwindCSS so you are not required to make changes in global CSS. use utility based classes for navbar and remove the CSS from global.css.
Commit those changes.

**Review by Deepu178 — CHANGES_REQUESTED** [review]

File: src/styles/globals.css (hunk 1 above)
Comment (ujjawaltyagii) [reply src/styles/globals.css]: done !!

**Comment by ujjawaltyagii** [comment]
@Deepu178 @J-Harsh  please review this PR

**Review by Deepu178 — APPROVED** [review]

**Comment by Deepu178** [comment]
image
It doesn't look same on my device as you updated in screenshot and I would suggest you to take a look, and make the changes as required.

**Comment by ujjawaltyagii** [comment]
@Deepu178 There was branch issue while commit but done with the changes now ig it looks good !

**Comment by Deepu178** [comment]
let's go for the merge🚀
@ujjawaltyagii Thankyou for making contribution in vibey.

**Review by Deepu178 — APPROVED** [review]

**Comment by ujjawaltyagii** [comment]
@Deepu178 please add label/ level and assignee to this PR 🙏 so that it can count for my gssoc entry

**Pull request merged**